src/app/api/tasks/tasks.service.ts

```diff
@@ -159,6 +163,8 @@ export class TasksService extends BaseService {
     })
     if (!task) throw new APIError(httpStatus.NOT_FOUND, 'The requested task was not found')
 
+    task.body = task.body && (await replaceImageSrc(task.body, this.getSignedUrl))
```

Comment:
what is `task.body` is undefined, what will this assignment resolve to? are we handling that case?

Reply:
If task is undefined, the function will throw error. task.body is nullable, so if task.body is null, i dont think we need any replacements.